src/actions/receive-profile.js

```diff
@@ -1017,6 +1022,36 @@ export async function doSymbolicateProfile(
   dispatch(doneSymbolicating());
 }
 
+export async function retrievePageFaviconsFromBrowser(
+  dispatch: Dispatch,
```

Comment:
mmm I see (and _I_ was the one who implemented `doSymbolicateProfile` initially so I can't blame anyone else :-) )

The difference I see though, is that `doSymbolicateProfile` starts the symbolication proces which can take long, while `retrievePageFaviconsFromBrowser` is a more direct process. But I admit the difference is thin.

I guess seeing that we pass `dispatch` is good enough to know that actions may be dispatched.

Reply:
This makes me think that we might want to run `doSymbolicateProfile` and `retrievePageFaviconsFromBrowser` in parallel. Or at least do `retrievePageFaviconsFromBrowser` first, because it's likely much faster than the symbolication, and it may happen that users share their profile before the symbolication ends.